distribution/docs/src/main/resources/content/_using/settings.adoc

```diff
@@ -0,0 +1,27 @@
+:title: Other Settings
```

Comment:
❓ Should this file be renamed to reflect that these are settings within the `Catalog UI` rather than settings in something like the `Admin Console`?

Reply:
can't hurt